### PR TITLE
remove unused cimport, parametrize test

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

File: pandas/_libs/writers.pyx [hunk 1]
```diff
@@ -16,8 +16,6 @@ cimport numpy as cnp
 from numpy cimport ndarray, uint8_t
 cnp.import_array()
 
-cimport util
```
Comment (jreback) [inline pandas/_libs/writers.pyx]: do you have a linter that catches this?

File: pandas/tests/sparse/test_libsparse.py [hunk 2]
```diff
@@ -596,22 +596,10 @@ def _check_case(xloc, xlen, yloc, ylen, eloc, elen):
...
-
-# too cute? oh but how I abhor code duplication
-check_ops = ['add', 'sub', 'mul', 'truediv', 'floordiv']
-
-
-def make_optestf(op):
-    def f(self):
-        sparse_op = getattr(splib, 'sparse_%s_float64' % op)
-        python_op = getattr(operator, op)
+    @pytest.mark.parametrize('opname',
```
Comment (jreback) [inline pandas/tests/sparse/test_libsparse.py]: worth making a fixture in conftest?

File: pandas/_libs/writers.pyx (hunk 1 above)
Comment (jbrockmendel) [reply pandas/_libs/writers.pyx]: We've discussed this a couple times and haven't found a good solution for this.  It's on my list to bring up with the group  at the sprint.

File: pandas/tests/sparse/test_libsparse.py (hunk 2 above)
Comment (jbrockmendel) [reply pandas/tests/sparse/test_libsparse.py]: I think this is a pretty specific set of strings that wouldn't get used often.  But I didn't look at this closely, just saw that the existing formatting was crying out to be updated.

File: pandas/_libs/writers.pyx (hunk 1 above)
Comment (jreback) [reply pandas/_libs/writers.pyx]: right, just asking if you found a soln!

File: pandas/tests/sparse/test_libsparse.py (hunk 2 above)
Comment (jreback) [reply pandas/tests/sparse/test_libsparse.py]: ok, that's fine

**Review by gfyoung — APPROVED** [review]

**Pull request merged**